src/ameba/rule/lint/unused_self_access.cr

```diff
@@ -32,10 +32,6 @@ module Ameba::Rule::Lint
       AST::ImplicitReturnVisitor.new(self, source)
     end
 
-    def test(source, node : Crystal::Self, in_macro : Bool)
-      issue_for node, MSG
-    end
```

Comment:
why do we remove this one? This not being traversed by `AST::ImplicitReturnVisitor` ?

Reply:
Because it's pointless. Among others, that's the reason we have specs - to test for regressions, and as you can see there are none failing...

Reply:
I agree, this should be fine